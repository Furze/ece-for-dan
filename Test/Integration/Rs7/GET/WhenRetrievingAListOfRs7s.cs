using MoE.ECE.Integration.Tests.Infrastructure;
using Xunit;
using Xunit.Abstractions;

namespace MoE.ECE.Integration.Tests.Rs7.GET
{
    // ReSharper disable once InconsistentNaming
    public class WhenRetrievingAListOfRs7s : SpeedyIntegrationTestBase
    {
        private const string Url = "api/rs7";

        public WhenRetrievingAListOfRs7s(
            RunOnceBeforeAllTests testSetUp,
            ITestOutputHelper output,
            TestState testState)
            : base(testSetUp, output, testState)
        {
        }

        protected override void Arrange()
        {
            If.A_rs7_has_been_created();
        }

        protected override void Act()
        {
            Api.Get(Url);
        }

        [Fact]
        public void ThenTheResponseShouldBeOk()
        {
            Then
                .TheResponse
                .ShouldBe
                .Ok<CollectionModel<Rs7Model>>();
        }

        [Fact]
        public void ThenTheResponseShouldContainSomeItems()
        {
            var response = Then
                .TheResponse
                .Content<CollectionModel<Rs7Model>>();

            response.Data.Length.ShouldBeGreaterThan(0);
            response.Pagination.Count.ShouldBeGreaterThan(0);
            response.Pagination.PageNumber.ShouldBe(1);
            response.Pagination.PageSize.ShouldBe(10);
        }
    }
}