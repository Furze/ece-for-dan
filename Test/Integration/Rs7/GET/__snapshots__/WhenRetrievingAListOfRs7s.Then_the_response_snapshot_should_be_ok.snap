{
  "Data": [
    {
      "Id": 177082,
      "BusinessEntityId": "02fad9ab-3e5f-493f-9352-72c1dd372202",
      "OrganisationId": 114895,
      "FundingPeriod": "July",
      "FundingYear": 2021,
      "FundingPeriodYear": 2020,
      "IsZeroReturn": false,
      "RollStatus": "ExternalNew",
      "ReceivedDate": null,
      "RevisionNumber": 1,
      "RevisionDate": "2020-11-12T14:58:59.8683596+13:00",
      "AdvanceMonths": [
        {
          "Id": 0,
          "MonthNumber": 7,
          "Year": 2020,
          "AllDay": null,
          "Sessional": null,
          "ParentLed": null
        },
        {
          "Id": 0,
          "MonthNumber": 8,
          "Year": 2020,
          "AllDay": null,
          "Sessional": null,
          "ParentLed": null
        },
        {
          "Id": 0,
          "MonthNumber": 9,
          "Year": 2020,
          "AllDay": null,
          "Sessional": null,
          "ParentLed": null
        },
        {
          "Id": 0,
          "MonthNumber": 10,
          "Year": 2020,
          "AllDay": null,
          "Sessional": null,
          "ParentLed": null
        }
      ],
      "EntitlementMonths": [
        {
          "Id": 0,
          "MonthNumber": 2,
          "Year": 2020,
          "Days": [
            {
              "DayNumber": 1,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 2,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 3,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 4,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 5,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 6,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 7,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 8,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 9,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 10,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 11,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 12,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 13,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 14,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 15,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 16,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 17,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 18,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 19,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 20,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 21,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 22,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 23,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 24,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 25,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 26,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 27,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 28,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 29,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            }
          ]
        },
        {
          "Id": 0,
          "MonthNumber": 3,
          "Year": 2020,
          "Days": [
            {
              "DayNumber": 1,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 2,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 3,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 4,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 5,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 6,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 7,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 8,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 9,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 10,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 11,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 12,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 13,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 14,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 15,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 16,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 17,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 18,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 19,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 20,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 21,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 22,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 23,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 24,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 25,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 26,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 27,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 28,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 29,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 30,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 31,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            }
          ]
        },
        {
          "Id": 0,
          "MonthNumber": 4,
          "Year": 2020,
          "Days": [
            {
              "DayNumber": 1,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 2,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 3,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 4,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 5,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 6,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 7,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 8,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 9,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 10,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 11,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 12,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 13,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 14,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 15,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 16,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 17,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 18,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 19,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 20,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 21,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 22,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 23,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 24,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 25,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 26,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 27,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 28,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 29,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 30,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            }
          ]
        },
        {
          "Id": 0,
          "MonthNumber": 5,
          "Year": 2020,
          "Days": [
            {
              "DayNumber": 1,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 2,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 3,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 4,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 5,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 6,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 7,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 8,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 9,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 10,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 11,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 12,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 13,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 14,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 15,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 16,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 17,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 18,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 19,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 20,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 21,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 22,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 23,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 24,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 25,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 26,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 27,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 28,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 29,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 30,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            },
            {
              "DayNumber": 31,
              "Under2": null,
              "TwoAndOver": null,
              "Hours20": null,
              "Plus10": null,
              "Certificated": null,
              "NonCertificated": null
            }
          ]
        }
      ],
      "IsAttested": null,
      "Declaration": null,
      "RequestId": "177082"
    }
  ],
  "Pagination": {
    "PageSize": 10,
    "PageNumber": 1,
    "Count": 1
  }
}
